presto-session-property-managers/pom.xml

```diff
@@ -75,6 +75,17 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-client</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.facebook.drift</groupId>
+            <artifactId>drift-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
curious why this was needed.
Where there any maven warnings/errors?

Reply:
Yes, there was a maven error for `drift-api` saying that it needs to be in provided scope:
```
[ERROR] Presto plugin dependency com.facebook.drift:drift-api must have scope 'provided'. It is part of the SPI and will be provided at runtime.

```